src/BrefKernel.php

```diff
@@ -41,7 +40,7 @@ public function getLogDir(): string
      *
      * @see https://github.com/brefphp/symfony-bridge/pull/37
      */
-    public function handle($request, $type = HttpKernelInterface::MASTER_REQUEST, $catch = true): Response
+    public function handle($request, $type = 1/* >=sf5.3 HttpKernelInterface::MAIN_REQUEST*/, $catch = true): Response
```

Comment:
Do we need to support Symfony 5.2 or can we bump to 5.4 for minimum?

Reply:
Symfony 5.2 and 5.3 are [EOL](https://symfony.com/releases#symfony-releases-calendar) since a few years. Let's bump it to min 5.4 :-)

Reply:
Yes let's bump 👍